docs/how_to_guides/how_to_load_policy_functions.ipynb

```diff
@@ -377,17 +421,15 @@
     }
    ],
    "source": [
-    "user_functions = \"gettsim.taxes.abgelt_st\"\n",
-    "user_functions"
+    "functions = abgelt_st\n",
+    "functions"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Combinations\n",
-    "\n",
-    "All previously shown methods can be combined to provide user functions for a user-defined tax and transfer system. Just combine all elements in a single list like the following example."
+    "Instead of importing modules, you can also provide an import statement as a string. The following would yield the same result as the solution before."
```

Comment:
Oh, I thought I should remove the conversion from strings to paths such that strings are always treated as import statements and paths to modules have to be `pathlib.Path`.

I thought loading via import statements is a nice-to-have because I always organize my research projects as packages and everything is importable. If you refactor something in this project, the import is usually fixed automatically whereas paths might have to be adjusted.

Reply:
Fair enough, just leave it as is then!